dora/src/language/impldefck.rs

```diff
@@ -195,13 +195,7 @@ impl<'x> ImplCheck<'x> {
 
         let parent = FctParent::Impl(self.impl_id);
 
-        let fct = FctDefinition::new(
-            self.sa,
-            self.file_id.into(),
-            self.namespace_id,
-            method,
-            parent,
-        );
+        let fct = FctDefinition::new(self.file_id.into(), self.namespace_id, method, parent);
```

Comment:
Why not open a separate PR for this refactoring? That we could land immediately and make this PR a bit smaller.